dev-python/ipython/ipython-3.2.1-r1.ebuild

```diff
@@ -15,7 +15,7 @@ SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="amd64 ~arm ~mips ppc ppc64 x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
+KEYWORDS="~amd64 ~arm ~mips ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
```

Comment:
"When upgrading, drop all existing keywords from arch to ~arch, and leave any existing ~arch keywords intact. [..] This also applies to revision bumps, not just to upstream version changes." [[1](https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades)]

Or do you mean I should leave the `ipython-3.2.1.ebuild` and add an additional `ipython-3.2.1-r1.ebuild`?